app/components/data-download.tsx

```diff
@@ -228,15 +224,35 @@ const DownloadMenuItem = ({
       loading={isDownloading}
       onClick={async () => {
         setIsDownloading(true);
+        urqlClient
+          .query(DataCubeObservationsDocument, {
+            locale,
+            iri: dataSetIri,
+            dimensions: null,
+            filters,
+          })
+          .toPromise()
+          .then(async (result: OperationResult<DataCubeObservationsQuery>) => {
```

Comment:
Using an await here would be clearer here.

Reply:
@bprusinowski I used Google Chrome Version 100.0.4896.75 (Official Build) (x86_64)
Still see the bug on your deployment link

Reply:
<img width="1659" alt="Screenshot 2022-04-13 at 14 37 49" src="https://user-images.githubusercontent.com/59613691/163181469-31d388b4-1d69-4a66-9153-c4370699ea75.png">

Reply:
I can't reproduce this on any of the browsers (Chrome, Safari, Firefox, Edge) – maybe it's a rare thing only happening on this version of Chrome on Intel chips? I don't really know how I can proceed with this at this point, maybe we'll see if it also occurs on TEST environment after deploying?